.github/workflows/docs.yml

```diff
@@ -0,0 +1,42 @@
+# GitHub action for the mmteb contributions
+# this way the CI related to the mmteb contributions will be separated from the linting CI
+
+name: documentation
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  create-table:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.RELEASE }}
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
Other workflow uses 3.10. Should we align at 3.8 or bump others to 3.10 as well?

Reply:
I don't think it matters. I would just use the most stable version for everything except for tests (3.11 would probably also be fine)